sup3r/preprocessing/data_handling.py

```diff
@@ -2166,6 +2314,30 @@ class DataHandlerH5SolarCC(DataHandlerH5WindCC):
     # wildcard format.
     TRAIN_ONLY_FEATURES = ('U*', 'V*', 'topography')
 
+    def __init__(self, *args, **kwargs):
+        """
+        Parameters
+        ----------
+        *args : list
+            Same positional args as DataHandlerH5
+        **kwargs : dict
+            Same keyword args as DataHandlerH5
+        """
+
+        args = copy.deepcopy(args)  # safe copy for manipulation
+        required = ['ghi', 'clearsky_ghi', 'clearsky_ratio']
+        missing = [dset for dset in required if dset not in args[1]]
+        if any(missing):
+            msg = ('Cannot initialize DataHandlerH5SolarCC without required '
```

Comment:
Can you explain this message? The sum stuff, in the context of computing an average, is confusing to me

Reply:
Yeah so the idea is that you cannot actually calculate a daily average clearsky ratio from the timeseries of clearsky ratios (weird). The correct way to calculate a daily average clearsky ratio is by taking the sum (or average) of the daily GHI and dividing by the sum (or average) of the daily clearsky GHI. Does that make sense?

Reply:
Oh yeah, that is weird. My confusion stemmed from thinking you could do the former.